### PR TITLE
Add in some tips and workarounds for problems (S1X)

### PR DESCRIPTION
**Pull request by Pawlicker**
I've added in some tips and workarounds from problems I've run into from hosting an S1X server myself, along with system requirements I've seen in the real world. I've also added in a link to the S1X logger script as this is needed for admin tools to work.

## Pull request type

 

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

### CODE REVIEW

File: support_s1x_server.html [hunk 1]
```diff
@@ -84,24 +84,38 @@ <h4>
...
+										server, with another 2GB per server 
+										instance you want to host. Each S1X 
+										server instance individually uses 2GB of 
+										RAM. </li>
+										<li>The 
+										<a
                                                 href="https://www.microsoft.com/en-us/download/details.aspx?id=5555">Visual
-                                                C++ 2010 Redistributable Package</a> and the <a
+                                                C++ 2010 Redistributable Package</a> and the 
+										<a
```
Comment (diamante0018) [inline support_s1x_server.html:98]: What is wrong with how formatting was before?

File: support_s1x_server.html [hunk 2]
```diff
@@ -116,16 +130,36 @@ <h4>
...
                                             and place the files in the server folder.</li>
+										<li>Open the redist folder and install 
+										all of the prerequisites.</li>
                                         <li>Run <code>!updateXLabs.bat</code> to update your server.</li>
                                         <!-- For now, Chase owns the repo used above, I can continue to host it or can transfer it to Xerxes or something. -->
                                         <li>Carefully review and edit the configuration settings file to your liking.
-                                            Give it a hostname, gametype, and remove/add maps that you want on the
-                                            rotate, etc.</li>
+                                            Give it a hostname, gametype, and remove/add maps that you want 
+										in the map rotation, etc.</li>
```
Comment (diamante0018) [inline support_s1x_server.html:139]: You could explain better what it means 'on the rotate' or' in the map rotation' to a user reading the guide. It doesn't feel like this bit is an improvement, unless your goal was to change the meaning of the sentence.

File: support_s1x_server.html [hunk 3]
```diff
@@ -116,16 +130,36 @@ <h4>
...
+											sure to allow connections <strong>
+											for both the inbound and outbound 
+											rules!</strong> Otherwise 
+											connections even within your own 
+											network will not work. </li>
+										</ul>
+										</li>
                                         <li>Run the start bat for the server you want and it should begin to load.</li>
+										<li>Test the server by opening up S1X, 
+										entering the console, and typing in
```
Comment (diamante0018) [inline support_s1x_server.html:159]: Elaborate `entering the console` It doesn't make sense to me. 
You could mention how to open the console, what to do if the tilde key is not present on your keyboard, etc...

File: support_s1x_server.html [hunk 4]
```diff
@@ -135,6 +169,13 @@ <h4>
                                             must be set in the map rotation. Example:
                                             <code>set sv_maprotation "gametype war map mp_refraction map mp_lab2 map mp_comeback"</code>.
                                         </li>
+										<li>Game logging (needed for IW4MAdmin 
+										and other administration tools)
+										<a href="https://github.com/Joelrau/S1x-IW6x-g_log-script">
+										requires a script to be installed</a> 
+										into the <code>s1x/scripts</code> 
+										folder.&nbsp; Without this script, 
+										logging simply will not work.</li>
```
Comment (diamante0018) [inline support_s1x_server.html:178]: I feel like the word 'simply' doesn't fit well in this sentence.

File: support_s1x_server.html [hunk 5]
```diff
@@ -143,9 +184,12 @@ <h4>
...
                                     <p>
-                                        IW4MAdmin partially supports S1x. Not all functions work but full support is
-                                        planned as the client is further developed. For downloads and more more
-                                        information, please visit <a
+                                        The latest builds of
+                                        IW4MAdmin partially support S1x. Not all functions work but full support is
+                                        planned as the client is further developed. 
+										The previously mentioned logging script 
+										has to be installed as well for this to 
+										function. For downloads and more information, please visit <a
```
Comment (diamante0018) [inline support_s1x_server.html:192]: Random formatting change I guess?

**Review by diamante0018 — CHANGES_REQUESTED** [review]

File: support_s1x_server.html (hunk 5 above)
Comment (Pawlicker) [reply support_s1x_server.html:192]: Sorry about that, I edited it in Expression Web and it changed the formatting for some reason.

File: support_s1x_server.html (hunk 2 above)
Comment (ChxseH) [reply support_s1x_server.html:139]: "in the rotation" makes more sense in this instance

**Pull request merged**